src/app.js

```diff
@@ -104,7 +104,11 @@ function createApp (config, ledgers, backend, routeBuilder, routeBroadcaster, ro
       currencyWithLedgerPairs: ledgers.getPairs(),
       backendUri: config.get('backendUri'),
       spread: config.get('fxSpread'),
-      getInfo: (ledger) => ledgers.getPlugin(ledger).getInfo()
+      getInfo: (ledger) => {
```

Comment:
I think giving access to the whole connected plugin is a bit ugly, the separation where only the info is accessible is a nice air gap.

I agree inserting the getBalance method into the info object is a bit ugly too. how about giving both `getInfo` and `getBalance`?